source/modes/GameMode.cpp

```diff
@@ -450,6 +458,7 @@ bool GameMode::mousePressed(const OIS::MouseEvent& arg, OIS::MouseButtonID id)
 
     if (id == OIS::MB_Middle)
     {
+        inputManager.mDragType=InputManager::rotateAxisXY;
```

Comment:
It's very nice that someone has offered me a help. I would gladly accept all the help for finishing this PR. Also if you could help with removing the starting camera tilt bug ...